From 1e5bb459bcb9746a9df08299e80167098766c946 Mon Sep 17 00:00:00 2001
From: Divya Theja <divyathe@codeaurora.org>
Date: Thu, 23 Aug 2018 18:00:19 +0530
Subject: [PATCH] Remove readprofile and brctl from busybox.links file

virtualizaton support needs brctl and readprofile provided by bridge-utils
and util-linux packages respectively. This change avoids busybox from
providing alternatives of these binaries.
---
 include/applets.src.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/include/applets.src.h b/include/applets.src.h
index dac83e7..72f9b49 100644
--- a/include/applets.src.h
+++ b/include/applets.src.h
@@ -85,7 +85,6 @@ IF_BASENAME(APPLET_NOFORK(basename, basename, BB_DIR_USR_BIN, BB_SUID_DROP, base
 IF_BBCONFIG(APPLET(bbconfig, BB_DIR_BIN, BB_SUID_DROP))
 IF_BEEP(APPLET(beep, BB_DIR_USR_BIN, BB_SUID_DROP))
 IF_BLKID(APPLET(blkid, BB_DIR_SBIN, BB_SUID_DROP))
-IF_BRCTL(APPLET(brctl, BB_DIR_USR_SBIN, BB_SUID_DROP))
 IF_CAL(APPLET(cal, BB_DIR_USR_BIN, BB_SUID_DROP))
 IF_CAT(APPLET_NOFORK(cat, cat, BB_DIR_BIN, BB_SUID_DROP, cat))
 IF_CATV(APPLET(catv, BB_DIR_BIN, BB_SUID_DROP))
@@ -280,7 +279,6 @@ IF_RDATE(APPLET(rdate, BB_DIR_USR_SBIN, BB_SUID_DROP))
 IF_RDEV(APPLET(rdev, BB_DIR_USR_SBIN, BB_SUID_DROP))
 IF_READAHEAD(APPLET(readahead, BB_DIR_USR_SBIN, BB_SUID_DROP))
 IF_READLINK(APPLET(readlink, BB_DIR_USR_BIN, BB_SUID_DROP))
-IF_READPROFILE(APPLET(readprofile, BB_DIR_USR_SBIN, BB_SUID_DROP))
 IF_REALPATH(APPLET(realpath, BB_DIR_USR_BIN, BB_SUID_DROP))
 IF_REFORMIME(APPLET(reformime, BB_DIR_BIN, BB_SUID_DROP))
 IF_RENICE(APPLET(renice, BB_DIR_USR_BIN, BB_SUID_DROP))
-- 
1.9.1

